Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.mm

```diff
@@ -787,9 +787,6 @@ - (void)didReplaceTab:(id<_WKWebExtensionTab>)oldTab withTab:(id<_WKWebExtension
     if (properties == _WKWebExtensionTabChangedPropertiesNone)
         return { };
 
-    if (properties == _WKWebExtensionTabChangedPropertiesAll)
-        return WebKit::WebExtensionTab::allChangedProperties();
-
```

Comment:
This wasn't mentioned in the commit message, was it just never used?

Reply:
I'll add it to the commit message. Feedback was that is not future proof and dangerous if more options are added later.